src/app/shared/map/map.component.ts

```diff
@@ -80,6 +85,25 @@ export class MapComponent {
 				this.placedMarker = event.coordinate;
 			});
 		}
+
+		this.map.getView().on('change:resolution', (event) => {
+			const zoom = this.map?.getView().getZoom();
+			const center = this.map?.getView().getCenter();
+			// Ensure that zoom and center change events are only emitted when they actually changed to prevent an event flood
+			if (zoom && Math.abs(zoom - this.lastZoomEvent) > 0.5) {
+				this.zoomChange.emit(zoom);
+				this.lastZoomEvent = zoom;
+			}
+
+			if (
+				center &&
+				Math.abs(center[0] - this.lastCenterEvent[0]) > 0.1 &&
+				Math.abs(center[1] - this.lastCenterEvent[1]) > 0.1
+			) {
+				this.centerChange.emit(center);
+				this.lastCenterEvent = center;
+			}
```

Comment:
this does not work yet, if I only change the center (move the map without zooming) I land back on the old center, anyways I would be nice that the map shows my own marker if im guessing, and all of the markers if I already solved by default. But lets not invest too much time into it probably

Reply:
https://github.com/uzh-ase-fs24/frontend/pull/44/commits/7cabe19fb17979794c5527bcd83cbc7f0b84367b